sass/_SimpleFilter.scss

```diff
@@ -0,0 +1,153 @@
+@import "bourbon/bourbon";
+@import "Variables";
+@import "../bin/sass/sprites";
+
+.coveo-filter-header-wrapper {
+  width: 100%;
+  margin:0 15px;
+  padding-left: 15px;
+  max-width: 1200px;
+  margin-left: auto;
+  margin-right: auto;
+  position: relative;
+  display: block;
+  @include flex-wrap(nowrap);
+  @include justify-content(flex-start);
+}
+.coveo-dropdown-background {
+  opacity: 0.9;
+}
+.CoveoSimpleFilter {
+  display: inline-block;
+  position: relative;
+  border: 1px #dadada solid;
+  background-color: white;
+  width: 160px;
+  &:hover {
+    background-color: $color-blueish-white-grey;
+  }
+}
+
+.coveo-simplefilter-selecttext {
+  max-width: 100px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.coveo-simplefilter-select {
+  color: #1D4F76;
+  cursor: pointer;
+  display: flex;
+  width: 159px;
+  justify-content: center;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  white-space: nowrap;
+  padding: 3px 19px;
```

Comment:
3 and 19 are pretty random. I have the feeling it's used to "center" something.

That's a bit of a red flag and it's possible that you could do that with other css rules.

For example `vertical-align : center` or something similar.

If it's not for that, please tell me :D